rasa_core_sdk/events.py

```diff
@@ -80,6 +80,15 @@ def ReminderScheduled(action_name, trigger_date_time, name=None,
     }
 
 
+# noinspection PyPep8Naming
+def ReminderCancelled(name, timestamp=None):
+    return {
+        "event": "cancel",
```

Comment:
let's name it explicitly
```suggestion
        "event": "cancel_reminder",
```